google-fit/src/appsscript.json

```diff
@@ -3,11 +3,11 @@
   "exceptionLogging": "STACKDRIVER",
   "dataStudio": {
     "name": "Google Fit",
-    "company": "Google",
+    "company": "Google Data Studio Developer Relations",
     "company_url": "https://fit.google.com/",
```

Comment:
We shouldn't have identical company names point to different company URLs. So In this case either remove companyUrl or make consistent with others: https://developers.google.com/datastudio/ or change company name to reflect company URL.